internal/pkg/cli/run_local.go

```diff
@@ -58,34 +61,41 @@ type containerOrchestrator interface {
 	Stop()
 }
 
+type hostFinder interface {
```

Comment:
nice separation of responsibility <img src="https://slackmojis.com/emojis/10804-meow_love/download" width=25px/>